ppa/local_settings.py.sample

```diff
@@ -113,3 +113,12 @@ LOGGING = {
 
     }
 }
+
+# URL to POST reports of content security policy violations to
+# https://github.com/mozilla/django-csp
+
+if not DEBUG: # don't report in debug mode
```

Comment:
Maybe we could have a configuration where dev doesn't set any endpoints, QA always has `CSP_REPORT_ONLY` set to `True` and points to the `/reportOnly` endpoint, and prod always has it set to `False`, and points to the `/enforce` endpoint?

That way we can view the reports in the report-ui.com interface and separate qa from prod using the enforced vs. report-only tags (just tested that this works - everything so far has been "report only" so I can filter it out).

Reply:
I think that makes the most sense. This would be pretty trivial to template the logic for in Ansible such that we could set QA / Prod as flexibly as we like, assuming it runs just fine without either setting (and sounds like that's the case).